### PR TITLE
[Metricbeat] Change visualization interval from 15m to >=15m

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
This PR is to change visualization interval from a value to `>=` the value to avoid `[tsvb] max_bucket exceeded` error.

Before this change:
81604431-b8959b80-9384-11ea-9e1f-b80f34bc83d6

Quote from Chris on why this is happening: 
_The problem is the interval on all those visualizations is set to 15m instead of >=15m. When you increase the time range, you will eventually hit the circuit breakers because the time series produces too many buckets._

After this change:
Screen Shot 2020-05-12 at 4 16 23 PM

### CODE REVIEW

File: x-pack/metricbeat/module/googlecloud/_meta/kibana/7/dashboard/Metricbeat-googlecloud-storage-overview.json [hunk 1]
```diff
@@ -413,14 +426,20 @@
...
         "description": "",
         "kibanaSavedObjectMeta": {
-          "searchSourceJSON": {}
+          "searchSourceJSON": {
+            "filter": [],
+            "query": {
+              "language": "kuery",
+              "query": ""
+            }
+          }
```
Comment (ycombinator) [inline x-pack/metricbeat/module/googlecloud/_meta/kibana/7/dashboard/Metricbeat-googlecloud-storage-overview.json:442]: Is this change necessary? I honestly don't know either way, just seems unrelated to the `interval` change.

File: x-pack/metricbeat/module/googlecloud/_meta/kibana/7/dashboard/Metricbeat-googlecloud-storage-overview.json [hunk 2]
```diff
@@ -501,7 +526,7 @@
             "default_timefield": "@timestamp",
             "id": "61ca57f0-469d-11e7-af02-69e470af7417",
             "index_pattern": "",
-            "interval": "5m",
+            "interval": "\u003e=5m",
```
Comment (ycombinator) [inline x-pack/metricbeat/module/googlecloud/_meta/kibana/7/dashboard/Metricbeat-googlecloud-storage-overview.json:529]: Is there a reason why some intervals are `>=15m` and others are `>=5m`?

**Comment by ycombinator** [comment]
In general, I'm wondering why we "hardcode" the intervals at all. Why not use `auto` and let Kibana figure out the optimal number of timeseries buckets to display based on the timepicker range?

**Comment by kaiyan-sheng** [comment]
@ycombinator The problem I see with `auto` interval is it does not connect the dots with lines in visualize. For example,
With `auto` interval:
Screen Shot 2020-05-13 at 9 13 30 AM

With hardcoded `>=5m` interval:
Screen Shot 2020-05-13 at 9 13 08 AM

**Comment by ycombinator** [comment]
Gotcha, thanks @kaiyan-sheng, let's stick with the "hardcoded" interval then.

File: x-pack/metricbeat/module/googlecloud/_meta/kibana/7/dashboard/Metricbeat-googlecloud-storage-overview.json (hunk 2 above)
Comment (kaiyan-sheng) [reply x-pack/metricbeat/module/googlecloud/_meta/kibana/7/dashboard/Metricbeat-googlecloud-storage-overview.json:529]: The visualizations with `>=15m` interval are showing the `Top N` data points. Because these metrics have an ingest delay of 10-minute and sample period of 5-minute. That's why the interval is hardcoded for `>=15m`. 

For other visualizations showing `Time Series` data, they have 2-minute ingest delays. With storage metricset recommends to use `5m` for collection period, I think `>=5m` visualization interval will work 😄

File: x-pack/metricbeat/module/googlecloud/_meta/kibana/7/dashboard/Metricbeat-googlecloud-storage-overview.json (hunk 1 above)
Comment (kaiyan-sheng) [reply x-pack/metricbeat/module/googlecloud/_meta/kibana/7/dashboard/Metricbeat-googlecloud-storage-overview.json:442]: Good point, I just changed on the dashboard and exported the json file. I will double check and if it's not necessary, I will remove it for sure.

Comment (kaiyan-sheng) [reply x-pack/metricbeat/module/googlecloud/_meta/kibana/7/dashboard/Metricbeat-googlecloud-storage-overview.json:436]: Removed!

File: x-pack/metricbeat/module/googlecloud/_meta/kibana/7/dashboard/Metricbeat-googlecloud-storage-overview.json (hunk 2 above)
Comment (ycombinator) [reply x-pack/metricbeat/module/googlecloud/_meta/kibana/7/dashboard/Metricbeat-googlecloud-storage-overview.json:523]: That makes sense. Thanks for explaining.

**Comment by kaiyan-sheng** [comment]
`searchSourceJSON` are all removed, thanks for reviewing!
